README.md

```diff
@@ -1,5 +1,10 @@
 # ![Graphene Logo](http://graphene-python.org/favicon.png) Graphene-Django
 
+> [!CAUTION]
```

Comment:
This syntax appears as follows (if you open this file via the top-right menu -> "View file", you'll be able to see this, unlike with the rich diff):

![image](https://github.com/user-attachments/assets/5c7e055f-ba2e-475e-9265-9c94007a0786)